### PR TITLE
pass context when not from global (e.g. ETP cart)

### PR DESCRIPTION
**Pull request by cliffordp**
[ETP-472]

[ETP-472]: https://moderntribe.atlassian.net/browse/ETP-472

### CODE REVIEW

File: src/views/v2/tickets/item/content.php [hunk 1]
```diff
@@ -50,8 +50,16 @@
...
+	[
+		'ticket'  => $ticket,
+		'post_id' => $post_id,
+	]
+);
 
-$this->template( 'v2/tickets/item/extra' );
+$this->template( 'v2/tickets/item/content/description', $context );
+
+$this->template( 'v2/tickets/item/extra', $context );
```
Comment (sc0ttkclark) [inline src/views/v2/tickets/item/content.php:65]: This shouldn't be necessary, if it is, we should fix it in a future sprint as part of a Common thing. This workaround shouldn't be needed/used.

File: src/views/v2/tickets/item/extra.php [hunk 2]
```diff
@@ -76,13 +76,14 @@
...
 <div <?php tribe_classes( $classes ); ?>>
 
-	<?php $this->template( 'v2/tickets/item/extra/price' ); ?>
+	<?php $this->template( 'v2/tickets/item/extra/price', $context ); ?>
 
-	<?php $this->template( 'v2/tickets/item/extra/available' ); ?>
+	<?php $this->template( 'v2/tickets/item/extra/available', $context ); ?>
 
-	<?php $this->template( 'v2/tickets/item/extra/description-toggle' ); ?>
+	<?php $this->template( 'v2/tickets/item/extra/description-toggle', $context ); ?>
```
Comment (sc0ttkclark) [inline src/views/v2/tickets/item/extra.php:87]: This shouldn't be necessary, if it is, we should fix it in a future sprint as part of a Common thing. This workaround shouldn't be needed/used.

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
src/views/v2/tickets/item/extra/available.php is good here, but the other two files changed includes code that shouldn't be necessary/used.

**Comment by cliffordp** [comment]
@sc0ttkclark both ET and ETP tests passing after removing what you said (this PR's most recent commit). There have been many changes to base branch and others' PRs so maybe the reason I did it resolved itself another way 👍 

ET passing:
image

ETP passing
image

**FYI: https://github.com/moderntribe/event-tickets-plus/pull/1087#issuecomment-726312170 was prior to this most recent commit. So both that PR and this PR should be good to merge.**

File: src/views/v2/tickets/item/extra/available.php [hunk 3]
```diff
@@ -53,6 +53,7 @@
```
Comment (sc0ttkclark) [inline src/views/v2/tickets/item/extra/available.php:53]: Needs since/version TBD update

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Needs since/version TBD update

**Pull request closed without merging**